src/assets/css/librarybrowser.css

```diff
@@ -671,6 +671,10 @@
     margin: 1em 0;
 }
 
+.layout-mobile .detailimg-hidemobile {
```

Comment:
Any reasoning of this class? Why are we using an specific class just for hiding it in mobile? Can't we use the same class that it's used in other layouts but set it as ``display: none`` when it has layout-mobile as a parent?

Reply:
It was a legacy class. I changed it to `.layout-mobile .detailImageContainer`, which is essentially the same.